### PR TITLE
fix EDR HTML breadcrumbs

### PR DESCRIPTION
**Pull request by tomkralidis**
# Overview
Fixes breadcrumbs and JSON link in EDR query HTML rendering.

# Related Issue / discussion
None

# Additional information
None
# Dependency policy (RFC2)

- [x] I have ensured that this PR meets [RFC2](https://pygeoapi.io/development/rfc/2) requirements

# Updates to public demo

- [x] I have ensured that breaking changes to the [pygeoapi master demo server](https://github.com/geopython/demo.pygeoapi.io) have been addressed
  - [ ] https://github.com/geopython/demo.pygeoapi.io/blob/master/services/pygeoapi_master/local.config.yml

# Contributions and licensing

(as per https://github.com/geopython/pygeoapi/blob/master/CONTRIBUTING.md#contributions-and-licensing)

- [x] I'd like to contribute [feature X|bugfix Y|docs|something else] to pygeoapi. I confirm that my contributions to pygeoapi will be compatible with the pygeoapi license guidelines at the time of contribution
- [x] I have already previously agreed to the pygeoapi Contributions and Licensing Guidelines

### CODE REVIEW

File: locale/en/LC_MESSAGES/messages.po
Comment (webb-ben) [inline locale/en/LC_MESSAGES/messages.po:1]: Any reason we are omitting Locations, Items, and  Instances?

File: pygeoapi/api/__init__.py [hunk 1]
```diff
@@ -80,6 +80,7 @@
 
 CHARSET = ['utf-8']
 F_JSON = 'json'
+F_COVERAGEJSON = 'json'
```
Comment (webb-ben) [inline pygeoapi/api/__init__.py:83]: Why is this not in `FORMAT_TYPES`?

File: pygeoapi/api/environmental_data_retrieval.py [hunk 2]
```diff
@@ -195,6 +198,31 @@ def get_collection_edr_query(api: API, request: APIRequest,
...
+        data['dataset_path'] = '/'.join(uri.split('/')[:-1])
+        data['collections_path'] = api.get_collections_url()
+
+        data['links'] = [{
+            'rel': 'collection',
+            'title': collections[dataset]['title'],
+            'href': data['dataset_path']
+        }, {
+            'type': 'application/prs.coverage+json',
+            'rel': request.get_linkrel(F_COVERAGEJSON),
```
Comment (webb-ben) [inline pygeoapi/api/environmental_data_retrieval.py:221]: EDR will not always be CovJSON

**Review by webb-ben — COMMENTED** [review]
Small questions. Another one is about the JSON-LD representation of EDR. At the moment the way the HTML pages are rendered, they do not account for query args when requesting the JSON-LD content. Ass such, any `/cube/...?bbox=...` HTML representation will request a JSON-LD representation without providing the bbox, producing an error in pygeoapi.

File: pygeoapi/api/__init__.py (hunk 1 above)
Comment (tomkralidis) [reply pygeoapi/api/__init__.py:83]: Adding to `FORMAT_TYPES` will overwrite F_JSON (same key) in the subsequent dict.  We can address in a subsequent PR.

File: pygeoapi/api/environmental_data_retrieval.py (hunk 2 above)
Comment (tomkralidis) [reply pygeoapi/api/environmental_data_retrieval.py:221]: Are we safe to assume, from a pygeoapi perspective, that CoverageJSON will be the baseline?

**Comment by tomkralidis** [comment]
> Small questions. Another one is about the JSON-LD representation of EDR. At the moment the way the HTML pages are rendered, they do not account for query args when requesting the JSON-LD content. Ass such, any `/cube/...?bbox=...` HTML representation will request a JSON-LD representation without providing the bbox, producing an error in pygeoapi.

Good catch @webb-ben!  JSON-LD HTML link updated with query parameters.

**Review by webb-ben — APPROVED** [review]

**Pull request merged**